tests/baselines/reference/destructuringInFunctionType.types

```diff
@@ -18,7 +18,7 @@ type F1 = ([a, b, c]) => void;
 >c : any
 
 type T2 = ({ a });
->T2 : { a: any; }
+>T2 : T2

```

Comment:
I can’t tell whether this makes sense because I can’t immediately deduce the rule for type readouts by reading the pre-existing examples below. `F2` gets reported as `F2` but `T3` prints out the tuple literal structure.

Reply:
Tuples are interned and don't have alias symbols, objects have alias symbols, so print as their alias if visible.